packages/eslint-config-terra/eslint.config.js

```diff
@@ -67,7 +67,7 @@ module.exports = {
       },
     },
     {
-      files: ['**/wdio/**/*-spec.*'],
+      files: ['**/wdio/**/*-spec.*', '**/full-stack/**/*-spec.*'],
```

Comment:
I forgot to include the spec files for full-stack on #640 so I'm adding it here before it is released.

Reply:
oops good catch, i forgot to list that in the issue